backend/cli_commands/SyncCommand.py

```diff
@@ -51,3 +52,24 @@ def sync_folder():
                 )
             except Exception as e:
                 logging.error(f"Error deleting mission '{name}': {e}")
+
+    # save metadata for each mission in the filesystem
+    for mission in db_missions:
+        tags = Mission_tags.objects.filter(mission=mission)
+        tag_data = []
+        for mission_tag in tags:
+            tag = Tag.objects.get(id=mission_tag.tag_id)
+            tag_data.append({"name": tag.name, "color": tag.color})
+        metadata = {
+            "location": mission.location,
+            "notes": mission.notes,
+            "tags": tag_data,
+        }
+        # save metadata to file inside mission folder
+        metadata_file = f"{mission.date.strftime('%Y.%m.%d')}_{mission.name}/{mission.name}_metadata.json"
+        metadata_file_path = storage.path(metadata_file)
+        with open(metadata_file_path, "w") as f:
```

Comment:
```suggestion
        with storage.open(metadata_file, "w") as f:
```
You need to use `storage.open` here if you want to use storages.\
And I think you then need to use `metadata_file` as the path.